tests/contrib/mariadb/test_mariadb.py

```diff
@@ -0,0 +1,388 @@
+import pytest
+import mariadb 
+from tests.utils import DummyTracer
+from tests.utils import assert_is_measured
+from tests.utils import assert_dict_issuperset
+from tests.utils import override_config
+from tests.utils import TracerTestCase
+from ddtrace.contrib.mariadb import patch
+from ddtrace.contrib.mariadb import unpatch
+from ddtrace import Pin
+from ddtrace import tracer as global_tracer
+from ddtrace.constants import ANALYTICS_SAMPLE_RATE_KEY
+from tests.utils import snapshot
+import unittest
+
+
+
+
+
+
+@pytest.fixture
+def tracer():
+    tracer = DummyTracer()
+    patch()
+    # Yield to our test
+    yield tracer
```

Comment:
this should probably be:

```python
try:
    yield tracer
finally:
  unpatch()
```

we want to be sure that no matter what we unpatch after every test case